apps/bar-chart/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/7.0.4/d3.min.js"></script>
+  </head>
+  <body>
+    <div class="main">
+      <div class="container">
+        <div id="title">United States GDP</div>
+        <div class="visHolder"></div>
+      </div>
+    </div>
+    <script src="public/script.js"></script>
```

Comment:
```suggestion
    <script src="script.js"></script>
```

This was not getting linked locally:
<img width="427" alt="Screen Shot 2022-09-30 at 8 56 34 AM" src="https://user-images.githubusercontent.com/20648924/193285826-fda0db5f-4ebe-4760-9fbf-4163ff9f1442.png">